dist/index.js

```diff
@@ -40977,7 +40983,7 @@ module.exports = {
  * SPDX-License-Identifier: MPL-2.0
  */
 
-const { core } = __nccwpck_require__(2186);
+const core = __nccwpck_require__(2186);
```

Comment:
This should also fix the throttling bug since it was being imported improperly before.